Correct admin password updating

dhageman opened the pull request:
##### SUMMARY

Corrects an issue with admin passwords failing to be updated due to shell escaping.  This aligns the operator with the logic in the normal installer.

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### ADDITIONAL INFORMATION
<!---
Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful.
-->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

Spredzy commented on roles/installer/tasks/initialize_django.yml:
```diff
@@ -18,10 +18,21 @@
+    pod: "{{ tower_pod_name }}"
+    container: "{{ ansible_operator_meta.name }}-task"
+    command: awx-manage update_password --username='{{ admin_user }}'' --password='{{ admin_password }}' --noinput
+  regiser: result
```
register :) @dhageman

dhageman replied:
You do have an eagle eye!  Thank you for the catch!

rooftopcellist commented on roles/installer/tasks/initialize_django.yml:
```diff
@@ -18,10 +18,21 @@
+    namespace: "{{ ansible_operator_meta.namespace }}"
+    pod: "{{ tower_pod_name }}"
+    container: "{{ ansible_operator_meta.name }}-task"
+    command: awx-manage update_password --username='{{ admin_user }}'' --password='{{ admin_password }}' --noinput
```
@dhageman It looks like CI is failing because there are two quotation marks here:

```
--username='{{ admin_user }}''
```

dhageman replied:
@rooftopcellist - Corrected now - thank you!

rooftopcellist commented on roles/installer/tasks/initialize_django.yml:
```diff
@@ -18,10 +18,21 @@
+    namespace: "{{ ansible_operator_meta.namespace }}"
+    pod: "{{ tower_pod_name }}"
+    container: "{{ ansible_operator_meta.name }}-task"
+    command: awx-manage update_password --username='{{ admin_user }}' --password='{{ admin_password }}' --noinput
```
Looks like `--noinput` on the update_password command causes issues.  Can you remove that @dhageman ?

It is still needed for the previous task that creates the user though.

dhageman committed:
Correct admin password updating

Corrects an issue with admin passwords failing to be updated due to shell escaping. This aligns the operator with the logic in the normal installer.

rooftopcellist commented:
Thanks for the fix @dhageman !
